lib/src/base-styles/_nx-forms.scss

```diff
@@ -81,3 +81,20 @@
   margin: 0 0 $nx-spacing-xl 0;
   padding: 0;
 }
+
+// Inline form group
+.nx-form-row--inline-form {
+  display: flex;
+
+  .nx-form-group--button-only {
+    @include container-horizontal;
+
+    align-items: flex-end;
+    display: flex;
+    flex-shrink: 1;
```

Comment:
What's this for?

Reply:
> What's this for?

It's gone now: https://github.com/sonatype/sonatype-react-shared-components/pull/164/commits/768f86743bae3a6b994fe0f05ae2f3ea359b6d52